ipam/fileIpam.go

```diff
@@ -11,18 +11,20 @@ import (
 	"runtime"
 	"strings"
 
+	"github.com/Azure/azure-container-networking/common"
 	"github.com/Azure/azure-container-networking/log"
 )
 
 const (
 	defaultLinuxFilePath   = "/etc/kubernetes/interfaces.json"
 	defaultWindowsFilePath = `c:\k\interfaces.json`
 	windows                = "windows"
-	name                   = "MAS"
+	masName                = "MAS"
```

Comment:
since you are importing common package, you don't require these fields, you can use
	OptEnvironmentMAS      = "mas"
	OptEnvironmentFileIPAM = "fileIpam"

Reply:
updated.